airflow/providers/presto/provider.yaml

```diff
@@ -62,7 +62,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.8.0
-  - apache-airflow-providers-common-sql>=1.3.1
+  - apache-airflow-providers-common-sql>=1.16.1
```

Comment:
Change to 1.17.0

Reply:
Yes. The version has been bumped already by #40751 and it would have to be 1.17.0 anyway as this is a new feature (MINOR is the middle number - the last one is PATCHLEVEL)